translations/nl-BE.yaml

```diff
@@ -347,4 +347,4 @@ attachments:
   type: Type
   decision: Besluit
   typePlaceholder: geen
-  regulatory: regelgevend
\ No newline at end of file
+  regulatory: regelgevend
```

Comment:
make sure to add this one to the english translation file as well

Reply:
done